packages/dev/docs/pages/react-aria/collections.mdx

```diff
@@ -257,17 +258,33 @@ let tree = useTreeData({
 function addPerson(name) {
   tree.append('People', {name});
 }
-
-<ListBox items={tree.items}>
-  {node =>
-    <Section id={section.name} items={node.children}>
-      <Header>{section.name}</Header>
-      <Collection items={section.children}>
-        {item => <ListBoxItem id={item.name}>{item.name}</ListBoxItem>}
-      </Collection>
-    </Section>
-  }
-</ListBox>
+const  onSubmit= e => {
+        e.preventDefault();
+        let data = Object.fromEntries(new FormData(e.currentTarget));
+        addPerson(data.personName)
+      }
+
+<>
+  <ListBox items={tree.items}>
+    {node =>
+      <Section id={section.name} items={node.children}>
+        <Header>{section.name}</Header>
+        <Collection items={section.children}>
+          {item => <ListBoxItem id={item.name}>{item.name}</ListBoxItem>}
+        </Collection>
+      </Section>
+    }
+  </ListBox>
+
+  <Form onSubmit={onSubmit}>
+    <TextField name="personName" isRequired>
+      <Label>Person Name</Label>
+      <Input />
+      <FieldError />
+    </TextField>
+    <Button type="submit">Submit</Button>
+  </Form>
+</>
 ```
```

Comment:
I appreciate wanting to make this example more complete, but I think we need to be careful not to overwhelm the reader with stuff that isn't super relevant. The point of the example it to show `useTreeData`, not `ListBox`, or `Form`, or `TextField`, etc. It mirrors the above example for `useListData`. For more details on using the methods, we have a link to the `useTreeData` docs. This page is just an overview so I think it's ok to not show everything in detail. So I think either we remove the `addPerson` function if people think it's confusing and leave the documentation of that to the `useTreeData` docs, or just leave it since it shows the general usage.